php/Slim4OtelDropIn/README.md

```diff
@@ -0,0 +1,33 @@
+# Distributed Tracing Example
+This example uses `docker-compose`, and illustrates the distributed tracing functionality of OpenTelemetry with Datadog's Drop In support enabled. An HTTP request to service-one will make multiple asynchronous HTTP requests, each of which is injected with a `traceparent` header, it includes examples of adding Attributes, Links and Events to spans.
+
+All trace data is sent to the [Datadog Agent](https://opentelemetry.io/docs/collector/), where they are forwarded to Datadog after you set your `DD_API_KEY`.
+
+The example is presented as a [slim framework](https://www.slimframework.com/) single-file application for simplicity, and uses Guzzle as an HTTP client. The same application source is used for all services.
+
+## Running the example
+```bash
+$ docker build -t php-service-with-dd-tracer .
+$ docker-compose run service-one composer install
+$ docker-compose up
+# in a separate terminal
+$ curl localhost:8000/users/otel
+```
+Update the `&otel-and-dd-common` section at the top of the `docker-compose.yml` to play around and match the screenshots below.
+
+## Screenshots
+### Opentelemetry
+`DD_TRACE_ENABLED: false & DD_TRACE_OTEL_ENABLED: false`
+![Opentelemetry](screenshots/distributed-otel-trace.png?raw=true "DD_TRACE_ENABLED: false & DD_TRACE_OTEL_ENABLED: false")
```

Comment:
Did you get traces even with both env var set to false? :eyes: I think I haven't understood something :( I was expecting... no traces? Am I wrong?

Reply:
For this one yes as I've set the collector env var for OTEL to sent to the DD Agent and what happens instead is that OTEL works alone and we only see OTEL Spans. Turning the tracer on then shows only DD spans and enabling it all got us both SDKs spans.

Reply:
Ohhhhh gotcha :+1: